policy-integrations/README.md

```diff
@@ -0,0 +1,72 @@
+# Border0 Policy Engine Integration Example
+
+This is an example project demonstrating how to integrate with the Border0 Policy Engine. This Flask application exposes a number of endpoints that can be called by the Border0 Policy Engine. It allows users to define their own logic and specify their own conditions for who should have access, using their own data and logic.
+
+## Endpoints
+
+The following endpoints are available:
+
+- `/fridayrule`: An endpoint that denies access on Fridays. Uses the requester's IP address to determine their current day of the week.
+
+- `/random`: This endpoint randomly generates a confidence score between 60 and 100. If the incoming request protocol is "http", it denies the request.
+
+- `/rainorshine`: This endpoint uses the requester's IP address to determine their current location and fetches the current weather data. If it's raining, the response will indicate so.
+
+- `/businesshours`: This endpoint checks if the request is made within business hours (between 9am and 6pm local time, Monday through Friday).
+
+Each endpoint requires a valid 'Authorization' header with a secret key. The default secret key is 'superSecret', but can be changed by setting the 'API_SECRET' environment variable.
+
+## Setup
```

Comment:
we should have a sample body for folks to paste in the body of their integration in this README. e.g.

```
{
   "ip":"${inject::ip}",
   "email":"${inject::email}",
   "protocol":"${inject::protocol}",
   "socket_id":"${inject::socket_id}",
   "socket_name":"${inject::socket_name}"
}
```

Reply:
I'll add a link to the blog and docs.